pkg/notes/notes.go

```diff
@@ -328,6 +328,19 @@ func (g *Gatherer) ListReleaseNotes() (*ReleaseNotes, error) {
 			}
 		}
 
+		toRemove := false
+		for _, filterKind := range g.options.FilterKinds {
+			if hasString(labelsWithPrefix(result.pullRequest, "kind"), filterKind) {
+				toRemove = false
+				break
+			}
+			toRemove = true
+		}
+
+		if toRemove {
+			continue
+		}
```

Comment:
We can use labels for that to avoid requiring `toRemove`:

```go
OUTER:
for _, item := range list.Items {
	for _, reserved := range reserved.Items {
		if reserved.ID == item.ID {
			continue OUTER
		}
		... do some other work ...
	}
	... do some other work ...
}
```

Reply:
This will work, but I'm not sure I really like labels. They tend to make code unreadable and harder to follow, but this is totally a personal opinion.

Reply:
ok i will do that, for me that sounds back to basic lol :D